google/cloud/alloydb/connector/async_connector.py

```diff
@@ -54,12 +54,14 @@ def __init__(
         quota_project: Optional[str] = None,
         alloydb_api_endpoint: str = "https://alloydb.googleapis.com",
         enable_iam_auth: bool = False,
+        user_agent: Optional[str] = None,
```

Comment:
nit:
```suggestion
        custom_user_agent: Optional[str] = None,
```

Reply:
I started with that actually and Jack and I've been debating it. Since the AlloyDB Go Connector uses "user agent," I think consistency is probably best here.